File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.Utils.Internal;
using DevExpress.XtraEditors;
using WindowsFormsApplication516.Properties;

namespace WindowsFormsApplication516 {

    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        MyDXLayeredWindow layeredWindow;
        MyBarDXLayeredWindow layeredWindow1;
        public Form1() {
            InitializeComponent();
        }
        protected override void OnShown(EventArgs e) {
            base.OnShown(e);
            layeredWindow = new MyDXLayeredWindow(simpleButton1, Resources.mail_16x16);
            layeredWindow.Show();

            layeredWindow1 = new MyBarDXLayeredWindow(barButtonItem1, Resources.mail_16x16);
            layeredWindow1.Show();

        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports DevExpress.Utils.Internal
Imports DevExpress.XtraEditors

Namespace WindowsFormsApplication516

    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Private layeredWindow As MyDXLayeredWindow
        Private layeredWindow1 As MyBarDXLayeredWindow
        Public Sub New()
            InitializeComponent()
        End Sub
        Protected Overrides Sub OnShown(ByVal e As EventArgs)
            MyBase.OnShown(e)
            layeredWindow = New MyDXLayeredWindow(simpleButton1, My.Resources.mail_16x16)
            layeredWindow.Show()

            layeredWindow1 = New MyBarDXLayeredWindow(barButtonItem1, My.Resources.mail_16x16)
            layeredWindow1.Show()

        End Sub
    End Class
End Namespace
